derivation/src/fpvm_evm/precompiles/mod.rs

```diff
@@ -0,0 +1,20 @@
+//! Contains the [`PrecompileProvider`] implementation that serves FPVM-accelerated OP Stack
+//! precompiles.
+//!
+//! [`PrecompileProvider`]: revm::handler::PrecompileProvider
+
+mod provider;
+pub(crate) use provider::OpFpvmPrecompiles;
+
+mod bls12_g1_add;
+mod bls12_g1_msm;
+mod bls12_g2_add;
+mod bls12_g2_msm;
+mod bls12_map_fp;
+mod bls12_map_fp2;
+mod bls12_pair;
+mod bn128_pair;
+mod ecrecover;
+mod kzg_point_eval;
+mod test_utils;
+mod utils;
```

Comment:
The suggested code is the original one taken from `kona/bin/client/src/fpvm_evm/precompiles/mod.rs`.
```suggestion
mod utils;

#[cfg(test)]
mod test_utils;
```

Reply:
Fixed by https://github.com/datachainlab/optimism-elc/pull/1/commits/fbb4b742258efedf829a62799024ce180e98df8a